expyfun/stimuli/_tracker.py

```diff
@@ -223,6 +237,10 @@ def respond(self, correct):
                     self._n_reversals += 1
                 if self._direction <= 0:
                     self._direction = 1
+        if self._repeat_limit == 'reversal':
```

Comment:
I'm with @Eric89GXL -- if hitting a min is going to bias the "true threshold" up I feel like changing the direction will make it worse.